Projekty/Projekt1/Gryszkiewicz_Chylak/Wstępny_Model/model_wstepny.ipynb

```diff
@@ -0,0 +1,2038 @@
+{
```

Comment:
<p>warto byłoby poinformować ile wierszy na tym etapie usunięto</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>